tensorflow/contrib/framework/BUILD

```diff
@@ -23,7 +26,41 @@ py_library(
         "python/ops/prettyprint_ops.py",
         "python/ops/variables.py",
     ],
+    data = [
+        ":python/ops/_variable_ops.so",
+    ],
     srcs_version = "PY2AND3",
+    deps = [
+        ":gen_variable_ops",
+    ],
+)
+
+tf_custom_op_library(
+    name = "python/ops/_variable_ops.so",
+    srcs = [
+        "ops/variable_ops.cc",
```

Comment:
Sort alphabetically (this has to be below the others) to get buildifier to stop complaining.